code-editors-wiki-article.md

```diff
@@ -0,0 +1,38 @@
+# Code Editors
+
+Source code editors are the programs that allow for code creation and 
+editing. Any text editor can be used to write code. But dedicated code
+editors offer many useful features, including: text completion, 
+markup coloring, and tabs/panes for side-by-side editing. These features can 
+be useful even to new users who don't know their way around all of the 
+advanced options. 
+
+There are many, many editors availble to download or use
+online, both paid and free. They offer many of the same core-features, but
+each have their own subtlties. Further research may help find the one that's
+right for you. The following editors are all cross-platform and 
+free to use or evaluate.
+
+## [Sublime Text](https://www.sublimetext.com/) <img src="https://upload.wikimedia.org/wikipedia/en/4/4c/Sublime_Text_Logo.png" alt="Sublime Text Icon" height=32px />
+![Sublime Text Screenshot](http://cache.filehippo.com/img/ex/2794__Sublime_text_2_1.png)
+
+Sublime Text is a very popular editor that has been around since 2008. There 
+are many options and extensions available, and a lot of help available online 
```

Comment:
Link to the package control site here - that bolsters your argument for many options and extensions.